ESS/v1/auxtel_ess01.yaml

```diff
@@ -1,6 +1,6 @@
 # Configuration for the AuxTel temperature sensor readout box
 host: auxtel-ess01.cp.lsst.org
 devices: [
-  {name: AuxTel-ESS01, channels: 4, device_type: FTDI, ftdi_id: AL05OBVR, sensor_type: Temperature},
-  {name: AuxTel-ESS02, device_type: Serial, serial_port: serial_ch_2, sensor_type: HX85BA},
+  {name: AuxTel-ESS01, channels: 4, device_type: FTDI, ftdi_id: AL05OBVR, sensor_type: Temperature, location: Unused:Air:Truss:M2},
+  {name: AuxTel-ESS02, device_type: Serial, serial_port: serial_ch_2, sensor_type: HX85BA, location: AzAxis},
```

Comment:
These lines are getting pretty long. I'll gently suggest that you switch to a different formatting (now or in the future), such as:
```
devices:
- name: AuxTel-ESS01
  channels: 4
  device_type: FTDI
  ftdi_id: AL05OBVR
  sensor_type: Temperature
  location: Unused:Air:Truss:M2
- name: AuxTel-ESS02
  device_type: Serial
  serial_port: serial_ch_2
  sensor_type: HX85BA
  location: AzAxis
```
Some people prefer to also indent the list items 2 spaces, e.g.:
```
devices:
  - name: AuxTel-ESS01
    channels: 4
    device_type: FTDI
    ftdi_id: AL05OBVR
    sensor_type: Temperature
    location: Unused:Air:Truss:M2
  - name: AuxTel-ESS02
    device_type: Serial
    serial_port: serial_ch_2
    sensor_type: HX85BA
    location: AzAxis
```
We seem to have yaml files formatted with and without the extra indentation, so I don't think we have a standard.

Reply:
I modified all yaml files to the different formatting. I also replaced all `:` for the AuxTel temperature sensor to `,`.